### ISSUE TITLE
Plex Versions not shown as duplicate

### ISSUE THREAD

**Issue by wivaku** [l3uddz/plex_dupefinder#25]
I create "Plex Versions", with the location: subfolder of original file.
Plex "get Info" shows both versions, ./plexdupes.py does not. I don't have anything in the SKIP_LIST.
Using Plex v1.16.5.1554

Example of movie and Plex Version:
```
.../movies/my Movie (2019)/my.Movie.WEB-DL.DDP5.1.H.264-NTG English.mkv
.../movies/my Movie (2019)/Plex Versions/Optimized for Mobile/my Movie (2019).mp4
```

**Comment by rhyguy** [comment]
Thanks for this awesome piece of software.  I second this issue, although it might just be that I need to format what I'm doing propperly.  I attempted to add "Plex Versions" to the SKIP_LIST but it seems to be ignored. I'm not entirely clear on what the formatting should be for the skip list.  Is it a literal quoted string, does the space in "Plex Versions" need to be escaped..

**Comment by wivaku** [comment]
@rhyguy [adding it to SKIP_LIST](https://github.com/l3uddz/plex_dupefinder/issues/14) is supposed to *ignore* the Plex Versions. The opposite of what I'm looking for.

**Comment by rhyguy** [comment]
Looking at the code.  It seems the logic for the skip isn't fired off until after you make the selection to delete.  For instance, I added "/Plex Versions/" to my SKIP_LIST and then run plexdupes.  It searches and then finds a movie with an optimized version and flags it as a dupe.  If you go ahead and tell it to "b", keep the best, it then moved to the routine that checks to see if the path contains the string in the SKIP_LIST.  In this case it matches so it skips the file and spits out the following message.

Choose item to keep (0 = skip | b = best): b
	Keeping 34
	Removing 35
	Skipping removal of this item as there is a match in SKIP_LIST

Seems like the issue here is just that a more desirable flow would be to check for a match in the SKIP_LIST first and don't even display anything, just move on to the next one.

TL;DR:  False positive is throwing a warning and it will skip anyway..  Just hit 'b' once and see..

**Comment by wivaku** [comment]
I have the feeling we're talking about two different things.
In my case Plex has 2 versions of "my Movie", but ./plexdupes.py does not detect them / show them as duplicates. I tried it both with empty SKIP_LIST and `"SKIP_LIST": ["/Plex Versions/"],`

**Comment by rhyguy** [comment]
@wivaku You're right.  I reread your initial post and now it makes more sense. You're looking to include the comparison, not exclude it like I am.

Sorry to unintentionally hijack your ticket...  :-)
